pkgs/applications/misc/prusa-slicer/default.nix

```
@@ -1,37 +1,44 @@
-{ stdenv, lib, fetchFromGitHub, cmake, pkgconfig
-, boost, cereal, curl, eigen, expat, glew, libpng, tbb, wxGTK31
-, gtest, nlopt, xorg, makeDesktopItem
-, cgal_5, gmp, ilmbase, mpfr, qhull, openvdb, systemd
+{ stdenv, lib, fetchFromGitHub, cmake, copyDesktopItems, makeDesktopItem, pkgconfig, wrapGAppsHook
+, boost, cereal, cgal_5, curl, dbus, eigen, expat, glew, glib, gmp, gtest, gtk3, hicolor-icon-theme
+, ilmbase, libpng, mpfr, nlopt, openvdb, pcre, qhull, systemd, tbb, wxGTK31-gtk3, xorg
 }:
 stdenv.mkDerivation rec {
   pname = "prusa-slicer";
-  version = "2.2.0";
+  version = "2.3.0";
 
   nativeBuildInputs = [
     cmake
+    copyDesktopItems
     pkgconfig
```

```suggestion
    pkg-config
```

The current is an alias for the latter, and is deprecated.

---

Done :)